setup.sh

```diff
@@ -266,9 +266,9 @@ echo -n "Building exonerate..."
 (
   if [ -e $SETUP_DIR/$CURR_TOOL.success ]; then
     echo -n " previously installed ..."
-  else [ $INST_METHOD -eq 2 ]; then
+  elif [ $INST_METHOD -eq 2 ]; then
```

Comment:
Why `2`? You don't use `1` anywhere.  Please don't copy/paste without evaluating the intent.

Reply:
sorry should have been looked more carefully